spec/superform/rails/form_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe Superform::Rails::Form, type: :rails do
```

Comment:
I don't want to use mocking or specs in the test quite; it makes it harder to understand and these can be implemented more concretely with the existing suite.